pkg/backup/label_helpers.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2018 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package backup
```

Comment:
This can be in a more neutral package?

Reply:
Yes +1 for this :)

Reply:
Note that these helpers specifically construct selectors/list options based on the `velero.io/backup-name=<value>` label -- I think that's probably why @nrb put them in `pkg/backup`. What other package would you prefer? (personally i'm fine with them where they are, but I'm not too concerned about it).

Reply:
Yeah, what Steve said.

I'm fine with moving it and making a more generic function, but I this was meant to generate selection criteria specifically based on the backup name label.

Reply:
My reason for suggesting the package change was to make these functions more discoverable. How about along side the `GetValidName` function in the `label` package?

Reply:
and suggest the functions can be named `NewBackupLabelSelector` and `NewListOptionsForBackup`.

Reply:
That's fine with me. I had them there before, but liked having the package name provide the specificity. However, I don't have a strong preference, so I'll move them into the `label` package and get the PR updated.